support-frontend/build.sbt

```diff
@@ -6,7 +6,7 @@ packageSummary := "Support Play APP"
 
 libraryDependencies ++= Seq(
   "com.typesafe" % "config" % "1.4.2",
-  "com.gu" %% "simple-configuration-ssm" % "1.5.7",
+  "com.gu" %% "simple-configuration-ssm" % "1.7.0",
```

Comment:
without this update we had a 
```
java.lang.NoClassDefFoundError: software/amazon/awssdk/protocols/query/interceptor/QueryParametersToBodyInterceptor
 at software.amazon.awssdk.services.autoscaling.DefaultAutoScalingBaseClientBuilder.finalizeServiceConfiguration(DefaultAutoScalingBaseClientBuilder.java:60)

```
This is because the odler SSM was compliled against an older AWS SDK v2, which wasn't binary compatible with the updated one I bumped in this PR.